components/credit-card/FinanceCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+
+export interface FinanceCardProps {
+    question: string,
+    answer: string,
+}
+
+const FinanceCard = ({question, answer}: FinanceCardProps) => {
+    const [color, setColor ] = useState("bg-gray-300")
+    const [borderColor, setBorderColor ] = useState("border-gray-500")
+    const [cardTitle, setcardTitle ] = useState("Click to Change")
+    
+    const onColorChangeRequested = () => {
+        if (color == "bg-gray-300") {
+            setColor("bg-green-300")
+            setBorderColor("border-green-500")
+            setcardTitle("debit")
+
+        } else if (color == "bg-green-300") {
```

Comment:
you should use === instead of ==

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness